azure-maven-plugin-lib/src/main/java/com/microsoft/azure/maven/telemetry/AppInsightHelper.java

```diff
@@ -54,9 +54,14 @@ public void trackEvent(String eventName, Map<String, String> customProperties, b
         if (!isEnabled) {
             return;
         }
-        final Map<String, String> properties = mergeProperties(customProperties, overrideDefaultProperties);
-        client.trackEvent(eventName, properties, null);
-        client.flush();
+        try {
+            final Map<String, String> properties = mergeProperties(customProperties, overrideDefaultProperties);
+            client.trackEvent(eventName, properties, null);
+            client.flush();
+        } catch (Exception ex) {
+            // swallow this exception
+            ex.printStackTrace();
```

Comment:
print stack or Log warning here?

Reply:
because the AI has some errors for sending the trace, print log so that user will notice it.